mycity/platforms/amazon/models/en_US.json

```diff
@@ -1228,6 +1228,15 @@
                         "is there an inclement weather emergency ",
                         "Is there a snow emergency"
                     ]
+                },
+                {
+                    "name": "FarmersMarketIntent",
+                    "slots": [],
+                    "samples": [
+                        "farmers markets today",
+                        "are there any farmers markets today",
+                        "farmers markets"
```

Comment:
I think we could use a few more examples. Here are some utterance that I tried and didn't work:
* where are farmers market
* where are farm stands
* where I can find a farmers market
* what farmer markets are open today

Reply:
I will add more utterances on the next push

Reply:
There are 15+ utterances now